### ISSUE TITLE
Some Informations are not looking correct for Income Statement

### ISSUE THREAD

**Issue by aturkkan94** [gadicc/node-yahoo-finance2#480]
# Bug Report

## Describe the bug

I have realized that some informations are not coming correct from the API. I checked for the other stocks also but it is the same.
If it is a bug and i m not making wrong something  could you please fix it.

Thanks...

Yahoo
![Screenshot_1](https://user-images.githubusercontent.com/66059183/170825991-f0092187-7299-4cc0-9e9c-f458ec7ae807.jpg)

API Response
![Screenshot_2](https://user-images.githubusercontent.com/66059183/170825993-4e8a7c3a-7c3f-4ccc-aaca-b7c2232d7c1a.jpg)

## Environment
Browser or Node: 
Node version (if applicable): 
Npm version: 
Browser verion (if applicable):
Library version (e.g. `1.10.1`):

**Comment by gadicc** [comment]
Hey, @aturkkan94, thanks for the great report.

Umm... from my early investigation on this, it looks like the mistake is on Yahoo's UI :sweat_smile:   I say this because if you view-source on the page you gave, you'll find the same data we're returning, and if you look at e.g. https://www.tradingview.com/symbols/BIST-SASA/financials-income-statement/?selected=total_revenue,net_income,oper_income, it matches these numbers too.  However, on https://www.investing.com/equities/sasa-polyester-income-statement, some of these particular fields had completely different numbers from both sources.  It would be great if we could find the company's own report, but I could only find their annual financials on their website.

If you figure out anything about this please let us know.  Otherwise leaving open for now to see if anyone else has anything to add.

Re some missing fields.  For the EBITDA on the example you gave, on the Yahoo's UI is shows as "-" so it makes sense it would be missing on the API query.  However, for other missing fields, we're tracking some in https://github.com/gadicc/node-yahoo-finance2/issues/476, let me know if we missing anything there.

**Comment by aturkkan94** [comment]
Yes if we check from Tradingview, the datas are looking correct.

Regarding EBITDA actually if we supply depreciation it is easy to calculate.
We will just make operatingIncome + depreciation and we can do it in UI...

EBITDA is important financial data actually that's why i highlighted it...

Thanks for your interest and kind reply...

**Comment by gadicc** [comment]
Thanks for that.  I'm not working in finance and don't personally consume this particular data, so it's great to have a user base who can give this kind of feedback.  And thanks again for the clear report.

**Comment by curwen** [comment]
Hey, I think I've found another example of this issue, although it's the other way round (UI correct, API wrong). HSBC's operating cash flow for 2021 was `104,312,000,000` as stated in their [annual report](https://www.hsbc.com/-/files/hsbc/investors/hsbc-results/2021/annual/pdfs/hsbc-holdings-plc/220222-annual-report-and-accounts-2021.pdf?download=1). This number is correct on both Yahoo's UI and investing.com. However, from the API's `cashflowStatementHistory` module, `totalCashFromOperatingActivities` incorrectly states the number as `8609000000`.
